deploy-web/src/queries/useProvidersQuery.ts

```diff
@@ -5,21 +5,21 @@ import { useSettings } from "../context/SettingsProvider";
 import { ApiUrlService, loadWithPagination } from "@src/utils/apiUtils";
 import { getNetworkCapacityDto, providerStatusToDto } from "@src/utils/providerUtils";
 import { PROVIDER_PROXY_URL } from "@src/utils/constants";
-import { RpcProvider } from "@src/types/provider";
+import { ApiProviderList, Auditor, RpcProvider } from "@src/types/provider";
 import { ProviderAttributesSchema } from "@src/types/providerAttributes";
 
-async function getProviderDetail(apiEndpoint: string, owner: string): Promise<RpcProvider> {
-  if (!owner) return {} as RpcProvider;
+// async function getProviderDetail(apiEndpoint: string, owner: string): Promise<RpcProvider> {
```

Comment:
:scissors:

Reply:
Actually, I will leave it there as it will be part of my next branch to complete the providers refactor